Classes/DashboardWidgets/IframeWidget.php

```diff
@@ -0,0 +1,87 @@
+<?php
+namespace TYPO3\CMS\Dashboard\DashboardWidgets;
+
+/*                                                                        *
+ * This script is part of the TYPO3 project - inspiring people to share!  *
+ *                                                                        *
+ * TYPO3 is free software; you can redistribute it and/or modify it under *
+ * the terms of the GNU General Public License version 2 as published by  *
+ * the Free Software Foundation.                                          *
+ *                                                                        *
+ * This script is distributed in the hope that it will be useful, but     *
+ * WITHOUT ANY WARRANTY; without even the implied warranty of MERCHAN-    *
+ * TABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General      *
+ * Public License for more details.                                       *
+ *                                                                        */
+
+use TYPO3\CMS\Dashboard\DashboardWidgetInterface;
+use TYPO3\CMS\Dashboard\Domain\Model\DashboardWidgetSettings;
+use TYPO3\CMS\Fluid\View\StandaloneView;
+
+class IframeWidget extends AbstractWidget implements DashboardWidgetInterface
+{
+
+    const IDENTIFIER = 1487642496;
```

Comment:
@MattiasNilsson that the widget identifier. It is used in 2 files so I used a constant for easy retrieval.